.docz/gatsby-node.js

```diff
@@ -0,0 +1,144 @@
+const get = require('lodash/get')
```

Comment:
Just out of curiousity, what is this file doing exactly?

Reply:
Yeah, everything in `.docz` is autogenerated by Docz. So, I'm not 100% sure.

Reply:
I see. I thought this was some regular gatsby thing. So that's good.